packages/stylelint-config/CHANGELOG.md

```diff
@@ -0,0 +1,246 @@
+<!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs. -->
+
+## Unreleased
+
+### Internal
```

Comment:
We either make it to ensure that release scripts trigger the major version bump:

```suggestion
### Breaking Changes
```

or we could try apply changes in `package.json` that I'm not 100% sure about:

```diff
+ "version": "18.0.0-prerelease",
- "version": "17.0.0",
```

Reply:
This is kind of related to the above re peerDependencies. If we change the versions this would be the breaking change.

Reply:
Thinking about it more, I guess the fact the name of the package changes is enough to call it a breaking change 😃